nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -266,18 +266,16 @@ SUBSYSTEM_DEF(star_system)
 	addtimer(CALLBACK(src, .proc/generate_anomaly), 15 SECONDS)
 
 /datum/star_system/proc/create_wormhole()
-	for(var/datum/star_system/S in SSstar_system.systems)
-		if(LAZYFIND(adjacency_list, S)) //We're already linked to that one. Skip it.
-			continue
+	var/datum/star_system/S = SSstar_system.systems[(rand(26))] //Pick a random system to put the wormhole in.

```

Comment:
```suggestion
	var/datum/star_system/S = SSstar_system.systems[(rand(SS_star_systems.systems.len))] //Pick a random system to put the wormhole in.

```

Reply:
No, that includes Abassi, Dolos, and DATA EXPUNGED, which we don't want.

Reply:
Unless we do want that.

Reply:
UNLESS????

Reply:
Helpful answer.